sdks/python/apache_beam/transforms/core.py

```diff
@@ -1358,6 +1358,15 @@ def expand(self, pcoll):
         *args, **kwargs)
 
   def to_runner_api_parameter(self, context):
+    # Note: Although this conditional seems identical to the one below it
+    # (if self.args or self.kwargs), there are cases where that condition
```

Comment:
Could you expand on what these cases are? Could we simply eliminate them instead of working around them?

Reply:
Actually, after investigating some more, it seems that the issue may be an actual problem with my change, which was hidden by this adjustment. I have to look into it more to see if I can fix it.